src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,70 @@
+import { FC } from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+import { TempTodo } from '../TempTodo';
+
+interface Props {
+  todos: Todo[];
+  isEdited: boolean;
+  onDelete: (id: number) => void;
+  todoId: number | null;
+  setTodoId: (id: number | null) => void;
+  tempTodo: Todo | null;
+}
+
+export const TodoList: FC<Props> = ({
+  todos,
+  isEdited,
+  onDelete,
+  todoId,
+  setTodoId,
+  tempTodo,
+}) => {
+  return (
+    <section className="todoapp__main">
+
+      <>
```

Comment:
Unnecessary Fragment '<>' can be removed, as it's not enclosing multiple elements.